lib/scrabble.rb

```diff
@@ -1,9 +1,37 @@
 class Scrabble
 
+  attr_reader :word
+
+  def initialize
+    @word = word
+  end
```

Comment:
Based on the interaction pattern, you don't need to pass an argument to `new`. It's possible that you would still want to have an `initialize` method, but in this case the result is that `word` will evaluate to `nil`, and because of that `@word` will always be assigned to `nil`.